Core/GCTestCase.h

```diff
@@ -79,6 +79,9 @@
 @interface GCMultipleCommitsRepositoryTests : GCMultipleCommitsRepositoryTestCase
 @end
 
+@interface GCUniqueCommitCollectionTests : GCMultipleCommitsRepositoryTestCase
```

Comment:
Yes it looks better in the Xcode UI, but that's not following the GitUp existing convention of how tests are structured, which is required :smile: